transforms/__testfixtures__/underscore-to-lodash-native.output.js

```diff
@@ -0,0 +1,5 @@
+// Test comment
+import {find} from 'lodash';
+[1, 2].forEach(num => num);
+const test = [1, 2, 3];
+const result = find(test, 2);
```

Comment:
Math.max is interesting, but would only work if no iteree function was provided. I can add some more tests, though they're all handled the same way, so it's a little redundant. But I'm okay with that.